parsers/CL_SING.py

```diff
@@ -203,6 +203,16 @@ def fetch_production(zone_key='CL-SING', session=None, target_datetime=None, log
         }
         production_mix_by_hour.append(production_mix)
 
+    # if production is the same for at least 3 in the last 48 hours, it's
+    # considered constant. Using round because of floating-point precision.
+    c = Counter([round(sum(v for _, v in e['production'].items()), 3)
+                 for e in production_mix_by_hour])
+    most_common = c.most_common(1)[0]
+    if most_common[1] > 3:
```

Comment:
Maybe we should add the number of occurrences to the error message.

Reply:
it is in the message, just two lines below

Reply:
Sorry I missed that, looks good to go.